Dockerfile

```diff
@@ -24,6 +24,11 @@ RUN useradd -u 911 -U -d $MEALIE_HOME -s /bin/bash abc \
     && usermod -G users abc \
     && mkdir $MEALIE_HOME
 
+RUN apt-get update && apt-get install -y \
+    # LDAP runtime dependencies
+    libldap-2.4-2 \
+    && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Is it necessary to add another layer and not include it as apart of the other installs? 

I'm not sure what the best practice is here.

Reply:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

While it's not as important as it used to be, Minimizing the number of layers is still on the Dockerfile best practices list, so this should probably be included with the rest of the installs